services/notify/schemas/v1/slack-request.yml

```diff
@@ -0,0 +1,32 @@
+$schema: "/schemas/common/metaschema.json#"
+title: Send Slack Message
+description: |
+  Request to send a message to a Slack channel. The most interesting field in
+  this request is the `blocks` field which allows you to specify advanced
+  display layout for messages. This is best understood via the Slack API
+  documentation.
+type: object
+properties:
+  channelId:
+    type: string
+    description: |
+      The unique Slack channel ID, such as `C123456GZ`.
+      In the app, this is the last section of the 'copy link' URL for a channel.
+  text:
+    type: string
+    description: |
+      The main message text. If no blocks are included, this is used as the
+      message text, otherwise this is used as alternative text and the blocks
+      are used.
+  blocks:
```

Comment:
I was a little stuck for these two - there is technically a specification for them but it would probably be bigger than the schema so far.